examples/showcase/firefox.py

```diff
@@ -0,0 +1,63 @@
+import re
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.path import Path
+import matplotlib.patches as patches
+
+# From: http://raphaeljs.com/icons/#firefox
```

Comment:
I only found this where they said icons were under MIT license: https://github.com/Sciss/RaphaelIcons